Recent Works Page & About Me page

sunga12 opened the pull request:
The Following pages:

- Recent Works
- About Me

have been Created

sunga12 committed:
Recent Works Page done

sunga12 committed:
Add About me Page

christianbiring1 commented on works.html:
```diff
@@ -0,0 +1,103 @@
+</head>
+<body>
+
+  <header>
+    <h1>My Recent Works</h1>
+  </header>
```
- [ ] After you put this section in the `index.html` file kindly make sure you grab this title in a section tag as well since it is the title of the section.

christianbiring1 commented on works.html:
```diff
@@ -0,0 +1,103 @@
+    <img class="gymfit" src="./images/Featured work.png" alt="Featured Work">
+    <h2 class="post-stories">Multi-Post Stories</h2>
+    <p class="feat-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+    <div class="lang-4">
+    <p class="css">css</p>
+    <p class="html">html</p>
+    <p class="bootstrap">bootstrap</p>
+    <p class="Ruby">Ruby</p>
```
- [ ] For semantic reasons, Kindly make sure the list of project technologies (e.g., `JavasScript`, `HTML`, and `CSS`) are a list `ul`, and each of them a `li` and not a generic paragraph since it's a list of elements

christianbiring1 commented on works.html:
```diff
@@ -0,0 +1,103 @@
+    <p class="bootstrap">bootstrap</p>
+    <p class="Ruby">Ruby</p>
+  </div>
+    <p class="see-project">See Project</p>
```
- [ ] The button `See project` is supposed to be a link `a` or `button` tag for semantic reasons, you don't have to grab it in a paragraph since it's not a text paragraph, and follow the button styling: `Active`, `Hover` and `Pressed` as shown on the design in Figma

christianbiring1 commented on works.html:
```diff
@@ -0,0 +1,103 @@
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p>
```
- [ ] The button `See project` is supposed to be a link `a` or `button` tag for semantic reasons, you don't have to grab it in a paragraph since it's not a text paragraph, and follow the button styling: `Active`, `Hover` and `Pressed` as shown on the design in Figma

christianbiring1 commented on works.html:
```diff
@@ -0,0 +1,103 @@
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p>
```
- [ ] The button `See project` is supposed to be a link `a` or `button` tag for semantic reasons, you don't have to grab it in a paragraph since it's not a text paragraph, and follow the button styling: `Active`, `Hover` and `Pressed` as shown on the design in Figma

christianbiring1 commented on works.html:
```diff
@@ -0,0 +1,103 @@
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p>
```
- [ ] The button `See project` is supposed to be a link `a` or `button` tag for semantic reasons, you don't have to grab it in a paragraph since it's not a text paragraph, and follow the button styling: `Active`, `Hover` and `Pressed` as shown on the design in Figma

christianbiring1 commented on works.html:
```diff
@@ -0,0 +1,103 @@
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p> 
```
- [ ] The button `See project` is supposed to be a link `a` or `button` tag for semantic reasons, you don't have to grab it in a paragraph since it's not a text paragraph, and follow the button styling: `Active`, `Hover` and `Pressed` as shown on the design in Figma

christianbiring1 commented on works.html:
```diff
@@ -0,0 +1,103 @@
+        </ul>
+
+      </div>
+      <p class="project-link"><a href="#">See Project</a></p>
```
- [ ] The button `See project` is supposed to be a link `a` or `button` tag for semantic reasons, you don't have to grab it in a paragraph since it's not a text paragraph, and follow the button styling: `Active`, `Hover` and `Pressed` as shown on the design in Figma

christianbiring1 commented on about.html:
```diff
@@ -0,0 +1,63 @@
+  <section class="about">
+    <h1 class="about-title">About me</h1>
+    <p class="about-me">Hello I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <p class="resume-link"><a href="#">Get My Resume</a></p>
```
- [ ] The button `See project` is supposed to be a link `a` or `button` tag for semantic reasons, you don't have to grab it in a paragraph since it's not a text paragraph, and follow the button styling: `Active`, `Hover` and `Pressed` as shown on the design in Figma

christianbiring1 reviewed (changes requested):
Hi @sunga12 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
![163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856](https://user-images.githubusercontent.com/97100091/212474545-c71cb810-e2ec-44f5-b752-e2a8f0b1714f.gif)


## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`

## Required Changes ♻️
- [ ] `IT'S A GREAT PRACTICE WORKING ON A COMPLEX PROJECT IN SINGLES ISOLATED COMPONENTS, HOWEVER, MAKE SURE AT THE END ALL YOUR CODES FROM THE about.html, works.html ARE PASTED IN THE index.html`
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

sunga12 committed:
Updated Portfolio according to 1st review

sunga12 commented:
Requirements:

- Move My recent works title to Section recent works section when merging all the pages together
- Change list of project technologies (Ruby, bootstrap, HTML, and CSS) to UL
- Change all 6 See Project p tags to buttons with "a" link on works page
- Change Get My Resume p tag to button with "a" link on About page

Requirements completed

- Moved My recent works title to Section recent works section when merging all the pages together
- Changed list of project technologies (Ruby, bootstrap, HTML, and CSS) to UL
- Changed all 6 See Project p tags to buttons with "a" link on works page
- Changed Get My Resume p tag to button with "a" link on About page

Owusu-Desmond reviewed (approved):
Hi @sunga12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media1.giphy.com/media/PudZiAbQDUEik/giphy.gif"/>

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
